codebase/base/src.lib/math/rmath.1.8/include/rmath.h

```diff
@@ -34,23 +38,28 @@
 #ifndef _RMATH_H
 #define _RMATH_H
 
-#ifndef _COMPLEX
+/*Need to use _COMPLEX_H to not
+ * override the complex.h definition*/
+#ifndef _COMPLEX_H
 struct complex {
     double x,y;
 };
 #define cabs(a) sqrt((a.x*a.x)+(a.y*a.y))
 #endif
 
-#define C 2.997e+8
+/*Speed of Light in m/s*/
+#define C 2.99792458e+8
 #define LN_TO_LOG 4.342944819
 
 #ifndef HUGE_VAL
 #define HUGE_VAL 1.0e36                /* default value of HUGE */
 #endif
  
-#ifndef PI
+/*PI constant*/
 #define PI 3.14159265358979323846
-#endif
+
+/*Earth's mean spherical radius in km*/
```

Comment:
Let's not rehash #268, but in it there was discussion that we should clarify which particular definition of Earth radius we use. The 6371.2km value comes from: https://en.wikipedia.org/wiki/Earth_radius#Average_distance_from_center_to_surface. If you search for mean radius of the Earth, you'll find there are a ton of definitions and this one is the first most common one (most common is R_1=(2*a+b)/3 = 6371.0km for the WGS84 reference ellipsoid).
```suggestion
/*Earth's mean spherical radius (average distance from center to the actual surface everywhere) in km*/
```